### PR TITLE
CFP and tickets

### PR DESCRIPTION
**Pull request by swapnilogale**
a. Added CFP info and turned on CFP flag and buttons
b. Added ticket info and turned on tickets flag and buttons

### CODE REVIEW

File: docs/_data/australia-2020-config.yaml [hunk 1]
```diff
@@ -17,12 +17,12 @@ buttons:
...
      #   link_absolute: /blog/moving-portland-prague-2020-online/
-      # - text: Buy a Ticket!
-     #   link: /tickets
+    - text: Buy a Ticket!
+      link: /tickets
     - text: Sponsor us
       link: /sponsors/prospectus
-    #  - text: Submit a Talk
-    #    link: /cfp
+    - text: Submit a Talk
```
Comment (mxsasha) [inline docs/_data/australia-2020-config.yaml:24]: Having three top buttons kinda breaks the layout - we only support two.
Screenshot 2020-07-21 at 11 19 24

**Review by mxsasha — COMMENTED** [review]
One minor nit.

Also, I have just created the Tito event so that the ticket sales work, but I need someone else to check it first. The tito event is not live yet.

Do you want to run a grant program for this conference? It is currently enabled on the ticket page, but the form links to the Prague form. I can set up a form, if you want to run it.

File: docs/_data/australia-2020-config.yaml (hunk 1 above)
Comment (swapnilogale) [reply docs/_data/australia-2020-config.yaml:24]: @mxsasha - I can check out Ti.to if you want. Can you assign it to my account in Ti.to?  

I'd like to use the grant program if that's ok - I realised we need to set up an Australian form for this. Can you set one up?

Comment (swapnilogale) [reply docs/_data/australia-2020-config.yaml:24]: @mxsasha - I'll turn off the Sponsor us button from the top menu.

Comment (mxsasha) [reply docs/_data/australia-2020-config.yaml:24]: I can't figure out how to give you access to the tito event - @thatdocslady any idea?
I've created the grant program form and updated the dates to something reasonable in 9a85f86.

Also, somehow flaglanding is still True in my latest checkout, and on the readthedocs build? But I saw a commit of you disabling it, must have snuck in again.

**Comment by swapnilogale** [comment]
@mxsasha can you help me with this error?

**Comment by mxsasha** [comment]
> @mxsasha can you help me with this error?

Fixed it by restarting that build - looks like an external brief glitch.

**Comment by swapnilogale** [comment]
@mxsasha @plaindocs looking good except 1 issue
https://writethedocs-www--1274.org.readthedocs.build/conf/australia/2020/opportunity-grants/ points to the correct AU form Sasha has set up, but 
https://www.writethedocs.org/conf/australia/2020/opportunity-grants/ from /tickets and /news/cfp-open still open the Prague one. 

What am I missing?

**Comment by plaindocs** [comment]
The PR isn't merged yet, so any changes to https://www.writethedocs.org/conf/australia/2020/opportunity-grants/  won't be live yet. Is that what you're asking?

**Comment by mxsasha** [comment]
Yes, this is confusing - we link to an absolute URL, therefore whatever is in the master branch. I just updated the URL from the ticket page to be relative, that makes more sense. News post URLs have to be absolute, because they get sent out in emails.

**Review by plaindocs — APPROVED** [review]
This looks solid to me.

**Pull request merged**